controllers/drcluster_controller.go

```diff
@@ -343,7 +343,8 @@ func (r DRClusterReconciler) processCreateOrUpdate(u *drclusterInstance) (ctrl.R
 
 	if reason, err := validateS3Profile(u.ctx, r.APIReader, r.ObjectStoreGetter, u.object, u.namespacedName.String(),
 		u.log); err != nil {
-		return ctrl.Result{}, fmt.Errorf("drclusters s3Profile validate: %w", u.validatedSetFalseAndUpdate(reason, err))
+		return ctrl.Result{Requeue: requeue},
```

Comment:
Do you want to do the same for line 351?

Reply:
Theoretically, you are right. We just didn't experience any failures in getDRClusterDeployedStatus yet. But I think it must be also covered.